src/examples/Example.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+"""
+ @file
+ @brief Python source file for openshot.py example
+ @author Jonathan Thomas <jonathan@openshot.org>
+ @author FeRD (Frank Dana) <ferdnyc@gmail.com>
+
+ @ref License
+"""
+
+# LICENSE
+#
+# Copyright (c) 2008-2019 OpenShot Studios, LLC
+# <http://www.openshotstudios.com/>. This file is part of
+# OpenShot Library (libopenshot), an open-source project dedicated to
+# delivering high quality video editing and animation solutions to the
+# world. For more information visit <http://www.openshot.org/>.
+#
+# OpenShot Library (libopenshot) is free software: you can redistribute it
+# and/or modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# OpenShot Library (libopenshot) is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with OpenShot Library. If not, see <http://www.gnu.org/licenses/>.
+
+import sys
```

Comment:
Remove this import, and you should pass the Codacy/PR Quality Review

Reply:
From my point of view, it was ready Sep 4 (https://github.com/OpenShot/libopenshot/pull/313/#discussion_r320969868). It produced correct results on my AMD and Intel HW encoders. The remaining work would be adding encoding presets to openshot-qt and consider improving HW encoder detection and selection (some people like me have both CPU and GPU encoders available).